using-kontena/install-nodes/cloud.md

```diff
@@ -0,0 +1,126 @@
+# Creating Kontena Cloud Nodes
+
+* [Prerequisites](#prerequisites)
+* [Create Kontena Node](#install-kontena-node)
+* [Restart Kontena Node](#restart-kontena-node)
+* [Terminate Kontena Node](#terminate-kontena-node)
+* [Update Kontena Node](#update-kontena-node)
+* [Advanced Usage](#advanced-usage)
+
+## Prerequisites
+
+* [Kontena CLI](/tools/cli.md) with the latest version of `cloud` plugin. If you don't have the plugin installed, you can install it with the: `kontena plugin install cloud` command.
+* [Kontena Cloud Platform](/using-kontena/platform.md). Kontena Cloud Nodes can be only attached to a Kontena Cloud Platform.
+
+## Create one or more Kontena Nodes
+
+```
+$ kontena cloud node create
+```
+
+## List Kontena Nodes
+
+List nodes in current organization:
+
+```
+$ kontena cloud node ls
+```
+
+List nodes in specific organization:
+
+```
+$ kontena cloud node ls --org <ORG>
+```
+
+## Open shell to a Kontena Node
+
+```
+$ kontena cloud shell <NAME>
+```
+
+## Restart a Kontena Node
+
+```
+$ kontena cloud node reboot <NAME>
+```
+
+## Terminate a Kontena Node
+
+```
+$ kontena cloud node terminate <NAME>
+```
+
+## Update a Kontena Node
+
+
+### Kontena Agent
+
+By default, Kontena Nodes will auto-update to the same version as Kontena Platform Master immediately if the `major` or `minor` version changes (`major.minor.x`). Kontena Nodes will do a patch level auto-update only when rebooted or systemd is restarted. This auto-update process can be forced by executing the following command in every Kontena Node host:
```

Comment:
By default, Kontena Nodes will auto-update to the same version as the Kontena Platform Master immediately if the `major` or `minor` version changes (`major.minor.x`). Kontena Nodes will do a patch level auto-update only when rebooted or systemd is restarted. This auto-update process can be forced by executing the following command in every Kontena Node host: